pyproject.toml

```diff
@@ -4,3 +4,10 @@ requires = [
     "wheel >= 0.30.0",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.black]
+target-version = ['py36']
```

Comment:
Shall we set `max-line-length` to 99 which is current configuration?

Reply:
Why `target-version` is Python 3.6?

Reply:
I would personally prefer 99 over 88. I think we have terminals wide enough to support this line length.

Reply:
@jhance I don't understand what you mean by terminals. What impact would terminal width have on code formatting?

@JelleZijlstra You're involved with Black right? What's your opinion on this.

Reply:
Yes, I'm a Black maintainer. I would also prefer using the defaults, but it's not a big deal if we use something else.

I think @jhance is referring to reading code in a terminal. It's bad if your lines are too long to fit in your terminal when using a terminal-based editor (e.g., vim).

Reply:
I would strongly prefer using mypy's existing line length

Reply:
I've set the line length to 99 😀

Reply:
Mypy will not support Python 3.6 anymore, so this should be 3.7.